examples/simple-authenticated-rest-api/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "commonjs",
+    "lib": ["es2020", "dom"],
```

Comment:
do you need `dom`? This doesn't look like it uses a DOM.

Reply:
This file is auto generated. It's the same as in all the other auto generated `tsconfig.json` files in these projects: 

- https://github.com/techfromsage/talis-cdk-constructs/blob/main/examples/simple-authenticated-api/tsconfig.json#L5
- https://github.com/techfromsage/talis-cdk-constructs/blob/main/examples/simple-lambda-worker/tsconfig.json#L5
- https://github.com/techfromsage/talis-cdk-constructs/blob/main/examples/simple-talis-cdk-stack/tsconfig.json#L5

I'm not sure about changing the auto generated files unless there is a strong reason to do so.